mesmer/mesmer_m/harmonic_model.py

```diff
@@ -152,65 +144,79 @@ def calculate_bic(n_samples, n_order, mse):
 
     """
 
-    n_params = n_order * 4 - 2
+    n_params = (
+        order * 4
+    )  # -2 because for order 0 the first two coefficients dissapear for sin(0)= 0
+    # removed this now because we don+t fit the beta0 and beta1
```

Comment:
```suggestion
    # NOTE: was -2 because for order=0 the first two coefficients disappear as sin(0) == 0
    # removed this now as we no longer fit beta0 and beta1
    n_params = order * 4
```

(I don't like if comments lead to code wraps)